### DISCUSSION TITLE
Adding additional domains and ip's to postal server

### DISCUSSION THREAD

**Discussion by Chris02468** [postalserver/postal#2720] [Help with using Postal]
I have some questions regarding the integration of additional IP addresses and domains into my existing Postal installation. Currently, my Postal server is configured as follows:

Hostname: [Postal.mydomain.com](http://postal.mydomain.com/)
IP Address: 1.1.1.1
Additional IPs added tp the server: 1.1.1.2, 1.1.1.3, 1.1.1.4
DNS Settings: Configured according to the documentation examples.

I have successfully added three servers (Server1, Server2, Server3), each with a dedicated domain and IP pool assigned to the Postal SMTP server. Now, as I proceed to set up DNS records for the additional domains and IPs, I am uncertain about the specific changes required in the configurations for [Postal.mydomain.com](http://postal.mydomain.com/) versus [mydomain1.com](http://mydomain1.com/) and [mydomain2.com](http://mydomain2.com/) etc..

Could you kindly provide guidance on the necessary modifications needed in the DNS records for these domains and IPs? I want to ensure that everything is correctly aligned and functioning as intended.

**Comment by willpower232** [comment]
I think you will need to have reverse DNS set up on all additional IPs as that is where the emails will come from but any bounces will probably still go to the regular psrp subdomain so as long as the SPF record refers to the IP Pool, that sounds roughly right. You can test with https://mail-tester.com to confirm they're sending okay.
